packages/optimize/src/Optimize.ts

```diff
@@ -121,6 +123,25 @@ const Optimize: IOptimize = {
       typeof onError === 'function' ? onError : () => {}
     );
   },  
+
+/**
+   * Dispatches an event for the Edge network extension to send an Experience Event to the Edge network with the display interaction data for the
+   * given list of Proposition offers.
+   * @param {Array<Offer>} offers - an array of Proposition offers
+   */
+  displayed(offers: Array<Offer>) {
+    RCTAEPOptimize.multipleOffersDisplayed(offers);
+  },
+
+/**
+ * Generates a map containing XDM formatted data for `Experience Event - OptimizeProposition Interactions` 
+ * field group from the provided list of PropositionOfferPair objects.
```

Comment:
Should we change the "PropositionOfferPair" to "Offer" to match the parameter type.